spec/std/file_utils_spec.cr

```diff
@@ -467,4 +467,179 @@ describe "FileUtils" do
       FileUtils.rm([path1, path2, path2])
     end
   end
+
+  describe "ln" do
+    it "should create a hardlink" do
+      path1 = "/tmp/crystal_ln_test_#{Process.pid}"
```

Comment:
Use `Tempfile.dirname` to get the path to the temporary directory instead of hardcoding it.  Consider using `Tempfile` itself to generate temporary "test" files instead.

Reply:
Do you think it would be a good idea to do this in another PR? `/tmp` is hardcoded all over this file, and might be good to update the tests all at once.

Reply:
Yes in that case it would be a good follow-up PR.